.clang-format

```diff
@@ -0,0 +1,12 @@
+BasedOnStyle: LLVM
+IndentWidth: 8
+UseTab: Always
+BreakBeforeBraces: Linux
+AllowShortIfStatementsOnASingleLine: false
+IndentCaseLabels: false
+
+# From CodingReadme
+TabWidth: 8
+ContinuationIndentWidth: 2
+ColumnLimit: 150
+BreakBeforeBraces: WebKit
```

Comment:
I think `BreakBeforeBraces: Linux` would be closer to the coding style we are using.